[CI] DockerTests test010Install failing

**Build Scans:**
- [elasticsearch-periodic-packaging #6739 / rhel-9_packaging-tests-unix](https://gradle-enterprise.elastic.co/s/75uganpr63h3m)
- [elasticsearch-periodic-packaging #6739 / rhel-9_packaging-tests-unix](https://gradle-enterprise.elastic.co/s/75uganpr63h3m)
- [elasticsearch-periodic-packaging #6739 / ubuntu-2204_packaging-tests-unix](https://gradle-enterprise.elastic.co/s/bi2vbybbb6xcg)
- [elasticsearch-periodic-packaging #6739 / ubuntu-2204_packaging-tests-unix](https://gradle-enterprise.elastic.co/s/bi2vbybbb6xcg)
- [elasticsearch-periodic-packaging #6739 / rocky-8_packaging-tests-unix](https://gradle-enterprise.elastic.co/s/7flazvibusavw)
- [elasticsearch-periodic-packaging #6739 / rocky-8_packaging-tests-unix](https://gradle-enterprise.elastic.co/s/7flazvibusavw)
- [elasticsearch-periodic-packaging #6739 / rocky-9_packaging-tests-unix](https://gradle-enterprise.elastic.co/s/mwltnps3okb7y)
- [elasticsearch-periodic-packaging #6739 / rocky-9_packaging-tests-unix](https://gradle-enterprise.elastic.co/s/mwltnps3okb7y)
- [elasticsearch-periodic-packaging #6739 / oraclelinux-9_packaging-tests-unix](https://gradle-enterprise.elastic.co/s/oj6tc2kpki4ee)
- [elasticsearch-periodic-packaging #6739 / oraclelinux-9_packaging-tests-unix](https://gradle-enterprise.elastic.co/s/oj6tc2kpki4ee)

**Reproduction Line:**
```
null
```

**Applicable branches:**
main

**Reproduces locally?:**
N/A

**Failure History:**
[See dashboard](https://es-delivery-stats.elastic.dev/app/dashboards#/view/dcec9e60-72ac-11ee-8f39-55975ded9e63?_g=(refreshInterval:(pause:!t,value:60000),time:(from:now-7d%2Fd,to:now))&_a=(controlGroupState:(initialChildControlState:('0c0c9cb8-ccd2-45c6-9b13-96bac4abc542':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:task.keyword,order:0,selectedOptions:!(),title:'GradleTask',type:optionsListControl),'4e6ad9d6-6fdc-4fcc-bf1a-aa6ca79e0850':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:className.keyword,order:1,selectedOptions:!(org.elasticsearch.packaging.test.DockerTests),title:'Suite',type:optionsListControl),'144933da-5c1b-4257-a969-7f43455a7901':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:name.keyword,order:2,selectedOptions:!(test010Install),title:'Test',type:optionsListControl)))))

**Failure Message:**
```
java.lang.AssertionError: Elasticsearch container did not start successfully.

ps output:
PID   USER     TIME  COMMAND
    1 elastics  0:00 /sbin/tini -- /usr/local/bin/docker-entrypoint.sh eswrapper
    7 elastics  0:15 /usr/share/elasticsearch/jdk/bin/java -Xms4m -Xmx64m -XX:+UseSerialGC -Djava.security.egd=file:/dev/urandom -Dcli.name=server -Dcli.script=/usr/share/elasticsearch/bin/elasticsearch -Dcli.libs=lib/tools/server-cli -Des.path.home=/usr/share/elasticsearch -Des.path.conf=/usr/share/elasticsearch/config -Des.distribution.type=docker -Des.java.type=bundled JDK -cp /usr/share/elasticsearch/lib/*:/usr/share/elasticsearch/lib/cli-launcher/* org.elasticsearch.launcher.CliToolLauncher
  275 elastics  0:52 /usr/share/elasticsearch/jdk/bin/java -Des.networkaddress.cache.ttl=60 -Des.networkaddress.cache.negative.ttl=10 -XX:+AlwaysPreTouch -Xss1m -Djava.awt.headless=true -Dfile.encoding=UTF-8 -Djna.nosys=true -XX:-OmitStackTraceInFastThrow -Dio.netty.noUnsafe=true -Dio.netty.noKeySe
[truncated]
```

**Issue Reasons:**
- [main] 2 consecutive failures in step ubuntu-2204_docker-cloud-ess_packaging-tests-unix
- [main] 2 consecutive failures in step opensuse-leap-15_docker-cloud-ess_packaging-tests-unix
- [main] 2 consecutive failures in step rhel-8_docker-cloud-ess_packaging-tests-unix
- [main] 2 consecutive failures in step oraclelinux-8_docker-cloud-ess_packaging-tests-unix
- [main] 2 consecutive failures in step almalinux-8_docker-cloud-ess_packaging-tests-unix
- [main] 2 consecutive failures in step ubuntu-2404_docker-cloud-ess_packaging-tests-unix
- [main] 2 consecutive failures in step rhel-9_docker-cloud-ess_packaging-tests-unix
- [main] 2 consecutive failures in step ubuntu-2004_docker-cloud-ess_packaging-tests-unix
- [main] 2 consecutive failures in step almalinux-9_docker-cloud-ess_packaging-tests-unix
- [main] 2 consecutive failures in step oraclelinux-9_docker-cloud-ess_packaging-tests-unix
- [main] 2 consecutive failures in step rocky-9_docker-cloud-ess_packaging-tests-unix
- [main] 2 consecutive failures in step rocky-8_docker-cloud-ess_packaging-tests-unix
- [main] 2 consecutive failures in step debian-12_docker-cloud-ess_packaging-tests-unix
- [main] 79 failures in test test010Install (23.3% fail rate in 339 executions)
- [main] 2 failures in step rhel-9_packaging-tests-unix (40.0% fail rate in 5 executions)

**Note:**
This issue was created using new test triage automation. Please report issues or feedback to es-delivery.

--- a/muted-tests.yml
+++ b/muted-tests.yml
@@ -390,9 +390,6 @@ tests:
 - class: org.elasticsearch.backwards.MixedClusterClientYamlTestSuiteIT
   method: test {p0=search/610_function_score/Random}
   issue: https://github.com/elastic/elasticsearch/issues/125010
-- class: org.elasticsearch.packaging.test.DockerTests
-  method: test010Install
-  issue: https://github.com/elastic/elasticsearch/issues/119441
 - class: org.elasticsearch.xpack.ilm.DataStreamAndIndexLifecycleMixingTests
   method: testGetDataStreamResponse
   issue: https://github.com/elastic/elasticsearch/issues/125083

--- a/qa/packaging/src/test/java/org/elasticsearch/packaging/util/docker/Docker.java
+++ b/qa/packaging/src/test/java/org/elasticsearch/packaging/util/docker/Docker.java
@@ -187,7 +187,7 @@ public class Docker {
                 Thread.sleep(STARTUP_SLEEP_INTERVAL_MILLISECONDS);
 
                 // Set COLUMNS so that `ps` doesn't truncate its output
-                psOutput = dockerShell.run("bash -c 'COLUMNS=3000 ps ax'").stdout();
+                psOutput = dockerShell.run("bash -c 'COLUMNS=4000 ps ax'").stdout();
 
                 if (psOutput.contains("org.elasticsearch.bootstrap.Elasticsearch")) {
                     isElasticsearchRunning = true;

